tracy-client/src/frame.rs

```diff
@@ -114,6 +114,30 @@ impl Client {
         }
         Frame(self.clone(), name)
     }
+
+    /// Emits an image of the current frame.
```

Comment:
```suggestion
    /// Emits an image of a frame.
```

Given that you can specify an offset to associate the image with an older frame.